lib/crystalball/rspec/runner.rb

```diff
@@ -69,44 +71,55 @@ def config_file
           file.exist? ? file : nil
         end
 
-        def build_prediction(out)
-          check_map(out)
+        def setup_logger
+          Crystalball::Logger.configure do |logger|
+            logger.log_level = config['log_level']
+            logger.color = config['color'].to_s == 'true' # ENV vars can only be string
```

Comment:
just define a proper getter for this variable in `lib/crystalball/rspec/runner/configuration.rb`

Reply:
Hm, nice, hadn't thought about that